builtin/credential/ldap/path_config.go

```diff
@@ -106,6 +106,11 @@ Default: cn`,
 				Default:     "tls12",
 				Description: "Maximum TLS version to use. Accepted values are 'tls10', 'tls11' or 'tls12'. Defaults to 'tls12'",
 			},
+			"deny_null_bind": &framework.FieldSchema{
+				Type:		 framework.TypeBool,
+				Default:	 false,
```

Comment:
Does it matter that all other bools for the ldap auth backend are `false` and this will be true?